FusionIIIT/applications/globals/migrations/0003_merge_20230406_1357.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.1.5 on 2023-04-06 13:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('globals', '0002_auto_20230323_2222'),
+        ('globals', '0002_auto_20230127_1451'),
+    ]
+
+    operations = [
+    ]
```

Comment:
@samay-rgb

Reply:
while making migration i did migration of all applications, so i pulled back older version to remove new migrations created for other applications. so kindly ignore it